rdpy/core/type.py

```diff
@@ -916,7 +925,26 @@ def writeType(self, value):
                 self.writeType(element)
             return
         value.write(self)
-        
+
+
+class ByteStream(BytesIO):
+    """
+    @summary:  ByteStream used to write all types, inherits BytesIO
+    """
+
+    def writeType(self, value):
+        """
+        @summary:  Call specific write on type object
+                    or iterate over tuple element
+        @param value: (tuple | Type)
+        """
+        if isinstance(value, tuple) or isinstance(value, list):
+            for element in value:
+                self.writeType(element)
+            return
```

Comment:
This might be a bit of a nitpicky style comment, but I prefer:
```python
if condition:
    do_thing_a()
else:
    do_thing_b()
```
over
```python
if condition:
    do_thing_a()
    return
do_thing_b()
```
because the former makes picking between alternatives clearer.